src/components/NavMenu.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react'
+
+import '../styles/home_nav.css'
+
+class NavMenu extends Component {
+  getDefaultProps() { return { isOpen: false }
+  }
+
+  render() {
+    if( this.props.isOpen ) {
+    return <div className="home-nav-dropdown-container">
```

Comment:
Indentation is still off here - the return statement needs to be indented under the if block, and indentation corrected further down in the function.